src/sys.c

```diff
@@ -106,6 +106,7 @@ struct VOLUME_PROFILE *search_volumes_targets(void)
                 }
                 if (list[n] != NULL) free(list[n]);
         }
+        free(list);
```

Comment:
Good catch. Although in good practice, I make sure that list is not null before I free to avoid a double freeing.

Reply:
> Good catch. Although in good practice, I make sure that list is not null before I free to avoid a double freeing.

Ok, I'll add this test asap to the `free()` calls.

Reply:
> Good catch. Although in good practice, I make sure that list is not null before I free to avoid a double freeing.

I read the docs, and found out that the pointer passed to `free()` is not nulled by default, I'll try to nullify it by hand.